helpers/addresses.py

```diff
@@ -823,6 +828,18 @@ def checksum_address_dict(addresses):
     "ftm": checksum_address_dict(ADDRESSES_FANTOM),
 })
 
+def get_registry(chain_id):
+    if chain_id == 1:
+        return registry.eth
+    elif chain_id == 137:
+        return registry.poly
+    elif chain_id == 56:
+        return registry.bsc
+    elif chain_id == 42161:
+        return registry.arbitrum
+    elif chain_id == 250:
+        return registry.ftm 
```

Comment:
nice, i like this a lot, @petrovska-petro and i were talking about doing this as well.

i dont see the harm in instead of passing `chain_id` to just test it against `chain.id` directly. this then also enables `from helpers.addresses import get_registry as contract_registry`